docs/shopify_app/webhooks.md

```diff
@@ -82,4 +82,38 @@ We have three mandatory GDPR webhooks
 
 The `generate shopify_app` command generated three job templates corresponding to all three of these webhooks.
 To pass our approval process you will need to set these webhooks in your partner dashboard.
-You can read more about that [here](https://shopify.dev/apps/webhooks/configuration/mandatory-webhooks).
\ No newline at end of file
+You can read more about that [here](https://shopify.dev/apps/webhooks/configuration/mandatory-webhooks).
+
+## EventBridge and PubSub Webhooks
+
+You can also register webhooks for delivery to Amazon EventBridge or Google Cloud Pub/Sub. In this case the `path` argument to needs to be of a specific form.
+
+For EventBridge, the `path` must be the ARN of the partner event source.
+
+```rb
+ShopifyApp.configure do |config|
+  config.webhooks = [
+    {
+      delivery_method: :event_bridge,
+      topic: 'carts/update',
+      path: 'arn:aws:events....'
+    }
+  ]
+end
+```
+
+For Pub/Sub, the `path` must be of the form `pubsub://[PROJECT-ID]:[PUB-SUB-TOPIC-ID]`. For example, if you created a topic with id `red` in the project `blue`, then the value of path would be `pubsub://blue:red`.
+
+```rb
+ShopifyApp.configure do |config|
+  config.webhooks = [
+    {
+      delivery_method: :pub_sub,
+      topic: 'carts/update',
+      path: 'pubsub://project-id:pub-sub-topic-id'
+    }
+  ]
+end
+```
+
+When registering for an EventBridge or PubSub Webhook you'll need to implement a handler that will fetch webhooks from the queue and process them yourself.
```

Comment:
@nelsonwittwer Unfortunately, I can't share publicly the code we use for the EventBridge dispatcher. But the way it works is by pulling events from the SQS and delegating them to webhook jobs using the same convention as HTTP webhooks. The example above from @kcamcam for `pub_sub` is similar.

It would be very helpful to create built-in `event_bridge` and `pub_sub` handlers for that gem. But that's a topic for another pull request. For now, it would be great if we could merge this tiny fix that will allow everyone to at least start using their own handlers with `shopify_app`.